qbosdk/apis/api_base.py

```diff
@@ -246,6 +253,8 @@ def _post_request(self, data, api_url):
         if response.status_code == 200:
             result = json.loads(response.text)
             return result
+        
+        logger.debug('Response for post request: %s', response.text)
 
```

Comment:
let's do logger.info for all non 2xx status code

logger.debug you can move it to 2xx block